holoviews/plotting/bokeh/element.py

```diff
@@ -933,14 +971,14 @@ def _init_tools(self, element):
         tools = []
         hover = False
         for key, subplot in self.subplots.items():
-                el = element.get(key)
-                if el is not None:
-                    el_tools = subplot._init_tools(el)
-                    el_tools = [t for t in el_tools
-                                if not (isinstance(t, HoverTool) and hover)]
-                    tools += el_tools
-                    if any(isinstance(t, HoverTool) for t in el_tools):
-                        hover = True
+            el = element.get(key)
+            if el is not None:
+                el_tools = subplot._init_tools(el, self.callbacks)
+                el_tools = [t for t in el_tools
```

Comment:
I guess ``OverlayPlot`` is merging the set of tools? Did it always have to do this or is this just for callbacks? If it is the latter, then I think this is another reason to have our own custom tool. :-)